lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -335,8 +339,20 @@ private Hover collectProperty(IPositionRequest request, String property) {
 			for (Entry<String, String> prop : allProps.entrySet()) {
 				String mavenProperty = prop.getKey();
 				if (property.equals(mavenProperty)) {
-					return new Hover(new MarkupContent("markdown", "Property: " + mavenProperty
+					Hover hover = new Hover(new MarkupContent("markdown", "Property: " + mavenProperty
 							+ MavenPluginUtils.LINE_BREAK + "Value: " + prop.getValue() + MavenPluginUtils.LINE_BREAK));
+					try {
+						Position propertyStart = new Position(request.getPosition().getLine(),
```

Comment:
This is code copied from MavenDefinitionParticipant. It should be factored into a function